### DISCUSSION TITLE
[keyball39]トラックボールが反応しない

### DISCUSSION THREAD

**Discussion by nekoze81** [Yowkees/keyball#471] [Q&A]
お世話になっております。

ビルドガイドに従い、keyball39を組み立てたのですが、トラックボールが反応しません。
カーソルが動かず、OLEDの移動量も変化がない状態です。

お手数おかけして申し訳ないのですが、原因切り分けのためのアドバイスいただけますでしょうか。

過去のdisucussionを参考に、
promicro-7Pコンスルーの導通確認を行いましたが、
添付画像の色付き箇所は問題なく導通しておりました。
また、promicroを左右入れ替えてみましたが、変化なしでした。

![tempImageYtDGhR](https://github.com/Yowkees/keyball/assets/75834489/501f0f8e-6e00-40ab-9888-3d8321d3bf02)
![tempImagenvOOon](https://github.com/Yowkees/keyball/assets/75834489/4b4f4a4e-e804-407d-ae83-e2973292b55d)
![tempImage2bvBhy](https://github.com/Yowkees/keyball/assets/75834489/cf342a6f-1e7d-4ab9-80e4-ae23ef45c491)
![tempImageszeQ9N](https://github.com/Yowkees/keyball/assets/75834489/d0f4b65b-1cd5-440d-90d7-fcb9309c196b)

![tempImageins4dj](https://github.com/Yowkees/keyball/assets/75834489/d8a19400-9e35-4ca9-9e00-3fc9cf391d2e)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

お写真を見る限りキレイに半田付けされており、問題となりそうな部分はありませんね。
ProMicro本体の半田付けかと思いましたが、左右のProMicroを入れ替えても復活しないということで、光学センサー等、部品単位での不具合があるのかも知れません。

現物確認での修理対応等をご相談したいので、お手数ですが下記白銀ラボウェブショップのお問合せよりご連絡いただければ幸いです。
https://shirogane-lab.net/inquiry

以上、よろしくお願いいたします。

**Reply by nekoze81** [reply]
@Yowkees 様

お返事ありがとうございます。

> ProMicro本体の半田付けかと思いましたが、左右のProMicroを入れ替えても復活しないということで、光学センサー等、部品単位での不具合があるのかも知れません。

ショップへの問い合わせの前に、念の為、右のProMicroの半田をやり直しましたところ、トラックボールが動くようになりました。
左のProMicroは試せていないのですが、半田が汚い箇所があるので、同じ理由で動かなかったのかもしれません。

おかげさまで、動くようになりましたこと、大変感謝しております。
この度は誠にありがとうございました。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
ご自身で半田付けを見直していただき、無事に動くようになったようで嬉しい限りです。
ぜひ末永くご愛用いただければ幸いです。今後ともどうぞよろしくお願いいたします。
